index.html

```diff
@@ -11,22 +11,186 @@
+            <div class="div">
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2 class="subtitle">Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><a class="button1">Ruby on rails</a></li>
+                    <li><a class="button1">css</a></li>
+                    <li><a class="button1">Java Script</a></li>
+                    <li><a class="button1">HTML</a></li>
+                </ul>
+                <button class="projects" type="button">See project</button>
+            </div>
+            <div class="div">
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2 class="subtitle">Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><a class="button1">Ruby on rails</a></li>
+                    <li><a class="button1">css</a></li>
+                    <li><a class="button1">Java Script</a></li>
+                    <li><a class="button1">HTML</a></li>
+                </ul>
+                <button class="projects" type="button">See project</button>
+            </div>
+            <div class="div">
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2 class="subtitle">Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><a class="button1">Ruby on rails</a></li>
+                    <li><a class="button1">css</a></li>
+                    <li><a class="button1">Java Script</a></li>
+                    <li><a class="button1">HTML</a></li>
+                </ul>
+                <button class="projects" type="button">See project</button>
+            </div>
+        
+            <div class="div">
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2 class="subtitle">Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><a class="button1">Ruby on rails</a></li>
+                    <li><a class="button1">css</a></li>
+                    <li><a class="button1">Java Script</a></li>
+                    <li><a class="button1">HTML</a></li>
+                </ul>
+                <button class="projects" type="button">See project</button>
+            </div>
+            
+            <div class="div">
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2 class="subtitle">Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><a class="button1">Ruby on rails</a></li>
+                    <li><a class="button1">css</a></li>
+                    <li><a class="button1">Java Script</a></li>
+                    <li><a class="button1">HTML</a></li>
+                </ul>
+                <button class="projects" type="button">See project</button>
+            </div>
+        </div>
+            </section>
```

Comment:
### OPTIONAL

-  Good job adding the border for each card in this section 💪🏿 

- [ ]  Please make the border the same color used in the Figma design 
- [ ] Please add the border radius for all cards  and buttons in this section 👍🏿